workshop-labs/docs/openshift101/00_overview.md

```diff
@@ -1,30 +1,25 @@
 # Overview
 Welcome!
 
-In these labs you will explore builds and deployments within the context of the Azure Redhat OpenShift Training Cluster.
+In these labs you will explore builds and deployments within the context of an Openshift 4.5 Cluster.
 These labs will focus on a single 2-tier application based on the popular open source chat utility, Rocket.Chat.
 
 ## Special Notes
 
-This training is being deliver in Openshift 4.3. The production Openshift cluster is currently in 3.11. 
+The training is being delivered in a production Openshift Environment.
 
 ## Course Format
 
 This course has been designed to be run _at your own pace_. The lab materials have been designed so that you generally follow the happy bath for a simple two-tier deployment. There will be many intentional bugs along the way that will have you encounter real-life troubleshooting techniques!
 
-The course length is approx. 2 weeks in length and during that time __touch points__ will be scheduled with the cohort to talk about special topics and to assist in troubleshooting. 
+The course length can vary (based on scheduled) but was designed to be run over approx. 2 weeks. During that time __touch points__ will be scheduled with the cohort to talk about special topics and to assist in troubleshooting. 
```

Comment:
```suggestion
The course length can vary (based on scheduled). It was designed to be run over a period of a few days rather than just a day or two to have time for isolated touchpoints as well as to provide time for discussion.  During that time __touch points__ will be scheduled with the cohort to talk about special topics and to assist in troubleshooting. 
```